packages/turf-interpolate/index.js

```diff
@@ -21,6 +31,7 @@ import { collectionOf } from "@turf/invariant";
  * @param {string} [options.property='elevation'] the property name in `points` from which z-values will be pulled, zValue fallbacks to 3rd coordinate if no property exists.
  * @param {string} [options.units='kilometers'] used in calculating cellSize, can be degrees, radians, miles, or kilometers
  * @param {number} [options.weight=1] exponent regulating the distance-decay weighting
+ * @param {BBox}   [options.bbox] assigned interpolation region
```

Comment:
Would be helpful to expand on this documentation a little. Maybe something additional like "If not provided here, defaults to bbox of input points."?

Reply:
Oh, I overlooked that. I’ll add it.